cpp/src/plasma/store.cc

```diff
@@ -259,6 +261,10 @@ PlasmaError PlasmaStore::CreateObject(const ObjectID& object_id, int64_t data_si
   entry->create_time = std::time(nullptr);
   entry->construct_duration = -1;
 
+#ifdef PLASMA_CUDA
+  entry->ipc_handle = ipc_handle;
```

Comment:
```suggestion
  entry->ipc_handle = result->ipc_handle;
```
Looks like `ipc_handle` wouldn't have been defined here.